Taskfile.yml

```diff
@@ -419,7 +419,7 @@ tasks:
         --proxy-addr=127.0.0.1:47017
         --mode=diff-normal
         --handler=pg
-        --postgresql-url='postgres://127.0.0.1:5433/ferretdb?search_path='
+        --postgresql-url='postgres://username:password@127.0.0.1:5433/ferretdb?search_path='
```

Comment:
For run-secured to work, what other better thing can I do?

Reply:
That target was supposed to be for the "old" authentication that does not need credentials in the PostgreSQL URI. Why are they needed?

Reply:
It's because postgres started on port 5433 is created from `postgres_secured` docker https://github.com/FerretDB/FerretDB/blob/main/docker-compose.yml#L28.
Unlike `postgres` docker which has `POSTGRES_HOST_AUTH_METHOD=trust`, `postgres_secured` doesn't have `trust` so username/password is required to connect to it.